### DISCUSSION TITLE
What URL do we fetch source tarballs from now?

### DISCUSSION THREAD

**Discussion by paperdigits** [geeqie/geeqie.github.io#4] [Q&A]
Previously, we could fetch a source tarball from http://geeqie.org/geeiqe-1.6.tar.xz but with the website refresh, that no longer works. What URL should we be using to fetch source now?

**Comment by lge** [comment]
On Thu, Jul 08, 2021 at 03:35:09PM -0700, Mica wrote:
> Previously, we could fetch a source tarball from
> http://geeqie.org/geeiqe-1.6.tar.xz but with the website refresh, that no
> longer works. What URL should we be using to fetch source now?

https://github.com/BestImageViewer/geeqie/tags
https://github.com/BestImageViewer/geeqie/releases

e.g.
https://github.com/BestImageViewer/geeqie/releases/download/v1.6/geeqie-1.6.tar.xz

    Lars

**Comment by caclark** [comment]
You may also go to https://github.com/BestImageViewer/geeqie, click on the "Code" button and download the zip file.
This file contains the latest sources. 
If you are compiling from sources you may as well use the latest - the more people who use the latest sources, the faster the bugs will be identified.
Of course there is then the problem of fixing the bugs :-(

**Comment by paperdigits** [comment]
Thanks Colin. I was updating the package in nixpkg, which doesn't accept "master" branches without good reason.
